serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-overview.adoc

```diff
@@ -0,0 +1,67 @@
+= Kogito Serverless Workflow Tools - Quarkus Dev UI
+
+Quarkus applications come with an interactive link:{quarkus_dev_ui_url}[Dev UI], that provides developers with a number of tools to help in the process of development. You can plug in these tools to a Quarkus application in the form of link:{quarkus_extension_url}[extensions]. The Kogito Serverless Workflow Tools is an extension, that provides a Console for you to view, manage and start Workflows Instances.
+
+.To add a dependency to your Quarkus project you need:
+* Quarkus {quarkus_version} is installed.
+* Install the Quarkus command line interface (CLI). For more information, see link:{quarkus_cli_url}[Installing the Quarkus CLI]
+
+Add the following command to install the Kogito Serverless Workflow Tools extension.
+
+.Command to install Kogito Serverless Workflow Tools extension
+[source,shell]
+----
+quarkus ext add org.kie.kogito:kogito-quarkus-serverless-workflow-devui
+----
+
+Executing the command adds the following _dependency_ to the _pom.xml_ of your project.
+
+.Kogito Serverless Workflow Tools dependency in pom.xml
+[source,xml]
+----
+<dependency>
+  <groupId>org.kie.kogito</groupId>
+  <artifactId>kogito-quarkus-serverless-workflow-devui</artifactId>
+  <version>${VERSION}</version>
+</dependency>
+----
+
+Another dependency for the Console is to add the _kogito-addons-quarkus-process-svg_ to provide the SVG diagram to the Consoles.
+
+.Command to install Kogito SVG add-on extension
+[source,shell]
+----
+quarkus ext add org.kie.kogito:kogito-addons-quarkus-process-svg
+----
+
+Executing the command adds the following _dependency_ to the _pom.xml_ of your project.
+
+.Quarkus Addons:Process-SVG dependency in pom.xml
+[source,xml]
+----
+<dependency>
+  <groupId>org.kie.kogito</groupId>
+  <artifactId>kogito-addons-quarkus-process-svg</artifactId>
+</dependency>
+----
+
+.The Quarkus application can be started by running the following command:
+[source,shell]
+----
+quarkus dev
+----
+
+The Quarkus Dev UI can be accessed by using `http://localhost:8080/q/dev/`
+
+.The Kogito Serverless Workflow Tools Extension in Quarkus Dev UI
+image::tooling/quarkus-dev-ui-extension/kogito-swf-tools-quarkus-dev-ui.png[]
+
+The Kogito Serverless Workflow Tools tab of the Dev UI shows the number of *Workflow Instances* available. Clicking on it will navigate you to the Console page.
```

Comment:
```suggestion
The Kogito Serverless Workflow Tools card above shows the number of *Workflow Instances* available. Clicking on it will navigate you to the Workflow Instances page.
```